tooling/support-ocp-version.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please move this file out of tooling directory. And how about to rename it to `supported-ocp-versions.json`.
Considering we have already used a json file, why not we just set the value as an array, like `["4.5", "4.6", "4.7", "4.8"]` instead of `"4.5 4.6 4.7 4.8"`?